app/models/user.rb

```diff
@@ -216,22 +216,10 @@ def miq_groups=(groups)
     self.current_group = groups.first if current_group.nil? || !groups.include?(current_group)
   end
 
-  def miq_group_ids
-    miq_groups.collect(&:id)
-  end
-
   def self.all_users_of_group(group)
     User.includes(:miq_groups).select { |u| u.miq_groups.include?(group) }
   end
 
-  def all_groups
-    miq_groups
-  end
```

Comment:
Can you verify this is not being dynamically used in the UI?  It's a common pattern for us to have tree based objects to have methods that start with `all_` to represent the entire subtree (also for counts on summary screens), so I'm wondering if the UI uses a dynamic method.